src/mpg123/mpg123.cc

```diff
@@ -265,8 +267,16 @@ static bool read_mpg123_info(const char * filename, VFSFile & file,
         {
             tuple.set_int(Tuple::Length, length);
             tuple.set_int(Tuple::Bitrate, aud::rdiv<int64_t>(8 * size, length));
+            if (aud::rdiv<int64_t>(8 * size, length) !=  s.info.bitrate) vbr = MPG123_VBR;
```

Comment:
This is just guessing.  It's easily broken by adding a large ID3 tag -- then the computed average bitrate is thrown off and a CBR file is reported by VBR.  I don't think we should do this.

Reply:
Isn't this also a problem that occurs when calculating the bitrate?